src/sparseml/yolov8/export.py

```diff
@@ -82,18 +80,12 @@
     help="cuda device, i.e. 0 or 0,1,2,3 or cpu. Device to run on",
 )
 @click.option(
-    "--datasets-dir",
+    "--dataset_path",
```

Comment:
Let's use --dataset-path, making it consistent w/ other arguments (dash instead of underscore)